### PR TITLE
[Hotfix] Comments Under Topics Nav

### PR DESCRIPTION
**Pull request by phillipahereza**
## Description

As suggested by Steen, add comments for the walk through

## Type of change

Please delete options that are not relevant.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: takwimu/templates/takwimu/_includes/profile/topic_detail.html [hunk 1]
```diff
@@ -23,11 +23,15 @@
...
-          </div> -->
+            <!-- TODO: Integrate with Zendesk -->
+      <div class="row pt-5 pb-5 text-dark">
+        {% if page.content_type.model == "profilesectionpage" %}
+              <p class=" pl-3">Add your voice to the discussion of {{ page.get_parent.title }}'s development context </p>
+        {% else %}
+              <p class=" pl-3">Add your voice to the discussion of {{ page.title }}'s development context </p>
+        {% endif %}
+              <a class="pl-3" href="" style="text-decoration: none;"><i class="fas fa-comments fa-2x"></i> &nbsp 89 comments</a>
+          </div>
```
Comment (DavidLemayian) [inline takwimu/templates/takwimu/_includes/profile/topic_detail.html]: How can we make this ready for launch?

Comment (phillipahereza) [reply takwimu/templates/takwimu/_includes/profile/topic_detail.html]: @DavidLemayian probably Zendesk community, I've been reading testing it and will have feedback in a few hours

**Comment by DavidLemayian** [comment]
@phillipahereza Please add screenshot.

**Comment by DavidLemayian** [comment]
@kilemensi We'll want to just reword "89 comments" to "Go to Forum" at this stage. @phillipahereza do you need help to do this?

**Pull request merged**